src/contracts/middleware/VoteWeigherBase.sol

```diff
@@ -59,7 +59,7 @@ contract VoteWeigherBase is VoteWeigherBaseStorage {
      * @notice This function computes the total weight of the @param operator in the quorum @param quorumNumber.
      * @dev reverts in the case that `quorumNumber` is greater than or equal to `quorumCount`
```

Comment:
This comment is no longer accurate

What's the reason for essentially duplicating the function without a modifier?

Reply:
oops, my mistake. good catch! b9071182